app/javascript/dashboard/routes/dashboard/settings/reports/components/overview/AgentTable.vue

```diff
@@ -130,7 +130,18 @@ export default {
       this.$emit('page-change', pageIndex);
     },
     getAgentInformation(id) {
-      return this.agents.find(agent => agent.id === Number(id));
+      const agentInformation = this.agents?.find(
+        agent => agent.id === Number(id)
+      );
+      if (!agentInformation) {
```

Comment:
@scmmishra Not sure why it's happening. I tried to replicate this issue, but it didn't work.

Reply:
Interesting, I understand, this can be a tricky one to debug. let's do this instead, instead of doing `this.agentMetrics.map` we can start with `this.agentMetrics.filter` and ensure that only those metrics are taken for which an agent is available. 

Let me know if this approach would work

Reply:
@scmmishra Done https://github.com/chatwoot/chatwoot/pull/9562/commits/99553a7135ced3fc0788f0bee3cd59171dff872f